### PR TITLE
FOLSPRINGB-26 Create v2.0.0 release notes

### PR DESCRIPTION
**Pull request by psmagin**
## Purpose
Create v2.0.0 release notes

### CODE REVIEW

**Review by dmtkachenko — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ Also, it is possible to specify logging level:
 `headers` - log all that `basic` and request headers
 `full` - log all that `headers` and request and response bodies
 
+***Note:*** *In case of you have async requests in your module (DeferredResult, CompletableFuture, etc.) then you should disable default logging for requests.*
```
Comment (natali-zaitseva) [inline README.md:62]: could you remove "of" to have "In case you have"

File: doc/release-notes.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+  @Value
+  class UserDto {
+
+    String id;
+    String username;
+  }
+}
+```
+ 4. Default for Folio log4j2 configurations introduced: `log4j2.properties` for plain and `log4j2-json.properties` for json format. So, you could remove log4j2 configurations from your module. See details in [README](../README.md#default-logging-format).
```
Comment (natali-zaitseva) [inline doc/release-notes.md:39]: Could you rephrase this to "The default configurations for Folio log4j2 are introduced"

File: doc/release-notes.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+ 5. Default logging for incoming and outgoing requests introduced. You should decide if you will use this mechanism. See details in [README](../README.md#logging-for-incoming-and-outgoing-requests).
```
Comment (natali-zaitseva) [inline doc/release-notes.md:40]: could you change "if you will use" to "if you use" or rephrase "You should decide whether to use this mechanism or not"

**Review by natali-zaitseva — APPROVED** [review]
